aggregator/src/aggregator.rs

```diff
@@ -2474,6 +2724,7 @@ impl VdafOps {
         vdaf: Arc<A>,
         req_bytes: &[u8],
         batch_aggregation_shard_count: u64,
+        peer_aggregator: Option<&PeerAggregator>,
```

Comment:
It's a shame that `handle_aggregate_share_generic` has to do gymnastics with the `Option<&PeerAggregator>` just so it can figure out how to get a collector HPKE config. More broadly, I think this layer of Janus shouldn't need to be aware of taskprov at all. What if somewhere higher up in the callstack, some code that is already dealing with the `dap-taskprov` header could figure out what collector HPKE config is relevant and then construct a `Task` with that?